libs/core/src/lib/text/text.component.ts

```diff
@@ -0,0 +1,91 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    Input,
+    ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+    selector: 'fd-text',
+    templateUrl: './text.component.html',
+    styleUrls: ['./text.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TextComponent {
+    /**
+     * Text for render
+     */
+    @Input()
+    text: string;
+
+    /**
+     * Max visible lines of text
+     */
+    @Input()
+    maxLines = null;
+
+    /**
+     * Property allowing browsers to render whitespace and tabs
+     */
+    @Input()
+    whitespaces?: boolean;
```

Comment:
I do not understand why some of these inputs are considered optional and others (such as maxLines) are not

Reply:
Same as above, I will change this.

Reply:
Changed